src/Workleap.Extensions.Mongo/MongoServiceCollectionExtensions.cs

```diff
@@ -37,6 +41,47 @@ public static MongoBuilder AddMongo(this IServiceCollection services, Action<Mon
         return new MongoBuilder(services);
     }
 
+    public static MongoBuilder AddCollectionConfigurations(this MongoBuilder builder, params Assembly[] assemblies)
```

Comment:
We could add some xml comments for the consumers.

Reply:
added